app/src/main/java/com/ensoft/imgurviewer/view/fragment/MediaPlayerFragment.java

```diff
@@ -270,13 +270,13 @@ protected void setFullscreenState( boolean enabled )
 		{
 			if ( enabled )
 			{
-				getActivity().setRequestedOrientation( ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE );
+				getActivity().setRequestedOrientation( ActivityInfo.SCREEN_ORIENTATION_SENSOR_LANDSCAPE );
 				
 				fullscreenOnOffView.setImageResource( R.drawable.ic_fullscreen_exit_white_48dp );
 			}
 			else
 			{
-				getActivity().setRequestedOrientation( ActivityInfo.SCREEN_ORIENTATION_SENSOR );
+				getActivity().setRequestedOrientation( ActivityInfo.SCREEN_ORIENTATION_USER );
```

Comment:
`SCREEN_ORIENTATION_SENSOR` is replaced with `SCREEN_ORIENTATION_USER` since SENSOR still rotates the screen even if the user disables auto-rotate in their system settings